.github/workflows/build.yml

```
@@ -1,3 +1,7 @@
+stages:
```

actionlint (1.7.7)

unexpected key "stages" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)